lib/rules/no-empty-headings.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+const AstNodeInfo = require('../helpers/ast-node-info');
+const Rule = require('./_base');
+
+const ERROR_MESSAGE = 'Heading (h1, h2, etc.) must contain accessible text content.';
```

Comment:
This message should probably be more general to also encompass an ARIA heading, or output a different message for that use case.

Reply:
I updated this message to include ARIA:heading role elements.